jobs/healing/pipeline-healing.js

```diff
@@ -229,23 +232,91 @@ async function getScopedSourceTriples( config, property, conceptSchemeUri, publi
   return reformatQueryResult(result);
 }
 
+function arrayToFile(array, file){
+  let fd = fs.openSync(file, "w");
+  for (let i=0; i<array.length; ++i){
+    fs.writeSync(fd, JSON.stringify(array[i]) + "\n")
+  }
+  fs.close(fd)
+  return file
+}
+function getTempFile() {
+  let file = "/tmp/healing-" + uuid();
+  return file;
+}
+function lines(filename) {
+  let retval = []
+  let rl = new readlines(filename)
+  let line
+  while ((line = rl.next())) {
+    line = line.toString()
+    if (line[0] === "<" || line[0] === ">") line = line.replace(/^./, "");
```

Comment:
I don't understand this

Reply:
this was code to strip the first character away for the output of the `diff` command, however it's been rendered useless since the `comm` command is used now